app/build.gradle

```diff
@@ -1,7 +1,10 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
-    id 'org.jetbrains.kotlin.android.extensions'
+    id 'org.jetbrains.kotlin.plugin.serialization'
+    id 'kotlin-kapt'
+    id 'dagger.hilt.android.plugin'
```

Comment:
For what?

Reply:
Let it be here. I need it in branch master